packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -396,6 +396,44 @@ export class WorkflowsController {
 		return true;
 	}
 
+	@Post('/:workflowId/archive')
+	@ProjectScope('workflow:delete')
+	async archive(req: WorkflowRequest.Delete) {
+		const { workflowId } = req.params;
+
+		const workflow = await this.workflowService.archive(req.user, workflowId);
+		if (!workflow) {
+			this.logger.warn('User attempted to archive a workflow without permissions', {
+				workflowId,
+				userId: req.user.id,
+			});
+			throw new ForbiddenError(
+				'Could not archive the workflow - you can only archive workflows owned by you',
+			);
```

Comment:
Workflows are owned by projects, not by users. Can we be more specific? Same below.

Reply:
We can, should I adjust the existing error message on `delete` endpoint which this was based on?

`'Could not delete the workflow - you can only remove workflows owned by you'`?

Reply:
Changed the texts on both the old delete endpoint and here. Also made them 403's to satisfy the AI bot reviews, even though they also occur on 404's.

Reply:
The existing one is also inaccurate. In any case, it's a nit - feel free to edit or disregard.